### PR TITLE
Adding demo command and updating README

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "lint": "npm run lint:wc && npm run lint:js",
     "lint:js": "eslint . --ext .js,.html test/**/*.js test/**/*.html demo/**/*.js demo/**/*.html",
     "lint:wc": "polymer lint",
-    "test:dev": "npm run lint && cross-env LAUNCHPAD_BROWSERS=chrome polymer test -p --skip-plugin sauce"
+    "test:dev": "npm run lint && cross-env LAUNCHPAD_BROWSERS=chrome polymer test -p --skip-plugin sauce",
+    "demo": "polymer serve --npm"
```
Comment (apalaniuk) [inline package.json]: I suppose `polymer serve` isn't smart enough to use the `npm` field in `polymer.json` to set this context as `polymer install`, obviating the need for the command?

**Comment by ChrisKraljevicD2L** [comment]
Apparently babel's polyfill [doesn't exist anymore.](https://travis-ci.com/Brightspace/d2l-richtext/builds/95435060#L523)

**Comment by apalaniuk** [comment]
> Apparently babel's polyfill [doesn't exist anymore.](https://travis-ci.com/Brightspace/d2l-richtext/builds/95435060#L523)

Seems like it should be referencing [@babel/polyfill/dist/polyfill{.min}.js](https://babeljs.io/docs/en/next/babel-polyfill.html#usage-in-browser)

File: README.md [hunk 2]
```diff
@@ -15,3 +15,13 @@ The `d2l-richtext` component takes in Annotated Html from a D2L Siren "richtext"
 	quicklink-consumer="d2l-richtext-quicklink-parent"
 ></d2l-richtext>
 ```
+
+## Demo
+
+Run:
+
+```
+npm run demo
```
Comment (apalaniuk) [inline README.md]: Presumably this is now wrong considering the script was removed? I think it should just be `polymer serve` (or `npx polymer serve` with a suitably new NPM version to run the `polymer-cli` dev dependency

**Pull request merged**